src/components/StatelessTransaction.js

```diff
@@ -59,6 +63,31 @@ const StatelessTransaction = props => {
       </React.Fragment>
   })
 
+  const gasPriceRangeDialogue = (
+    <React.Fragment>
+      <div className="mt-8">Gas Price: <span className="ml-4 text-700 text-sm">{(gasPrice / 1000000000).toFixed('9')} eth</span></div>
+
+      <input
+        className="mt-4"
+        type="range"
+        min="4"
```

Comment:
imo this should be able to go as low as 2 gwei.

Reply:
Roger.